metricbeat/module/mongodb/status/data.go

```diff
@@ -0,0 +1,241 @@
+package status
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+func eventMapping(status map[string]interface{}) common.MapStr {
+	errs := map[string]error{}
+	event := common.MapStr{
+		"version": mustBeString("version", status, errs),
+		"uptime": common.MapStr{
+			"ms": mustBeInteger("uptimeMillis", status, errs),
```

Comment:
The reason I couldn't use the ToInt stuff is that the stuff from mongo is already parsed in a `map[string]interface{}` so no conversions from strings are required. Very similar otherwise, though.

About the errs, I wanted to have the key missing in case of errors, rather than filled with the zero value.